src/InfluxDbClient.cpp

```diff
@@ -221,6 +225,8 @@ bool InfluxDBClient::init() {
               wifiClientSec->setCACert(_certInfo);
          }
 #endif    
+        if (_insecure)
+            wifiClientSec->setInsecure();
```

Comment:
`setInsecure` is not supported for ESP32. Move this to up to the conditioned compile part above.